api/views/session_views.py

```diff
@@ -224,6 +228,37 @@ def get(self, request, quiz_id):
         return JsonResponse({"quiz_sessions": quiz_sessions_data})
 
 
+@extend_schema(tags=["Session Management"])
+class QuizSessionLogView(APIView):
+    permission_classes = []
+
+    @extend_schema(
+        request=AddQuizSessionLogSerializer(),
+        responses={
+            201: OpenApiResponse(description="Log created and added successfully"),
```

Comment:
Here, we should specify the serializer, since we'd like Swagger to serve as the documentation of the input/output of these API endpoints.